content/operate/rc/databases/rdi/setup.md

```diff
@@ -0,0 +1,66 @@
+---
+Title: Prepare source database
+alwaysopen: false
+categories:
+- docs
+- operate
+- rc
+description: Prepare your source database and database credentials for Data integration.
+hideListLinks: true
+weight: 1
+---
+
+## Create new data pipeline
+
+1. In the [Redis Cloud console](https://cloud.redis.io/), go to your target database and select the **Data Pipeline** tab.
+1. Select **Create data pipeline**.
+1. Select your source database type. The following database types are supported:
```

Comment:
You also mention MariaDB and Aurora in the intro page. AFAIK, MariaDB is basically the same as MySQL, so I generally write MySQL/MariaDB together. Is Aurora specific to cloud RDI, with it being an Amazon product? Also, if we do support Aurora for this then do we need some extra instructions for it in the Prepare Source Databases section?

Reply:
@yaronp68 - Can you elaborate more on the Aurora inclusion? Is it the same as mySQL/mariaDB, or do we need to add more information as Andy said?